### PR TITLE
Fix movable background seekbar

### LINKED ISSUE #412: The orange bar in "power up info" shouldn't be user movable
The orange bar in the seekbar that shows which part of it is about the current trainer level is user movable if we try to drag it. It should not be moveable.

@oskomonzon -- you might be familiar enough with this to fix it quickly?

### PR DESCRIPTION
**Pull request by DJCrapsody**
Add empty on touch listener to prevent changes to seekbar thumb 1
Fix #412

### CODE REVIEW

File: app/src/main/java/com/kamron/pogoiv/Pokefly.java [hunk 1]
```diff
@@ -1071,16 +1071,17 @@ private void populateResultsBox(IVScanResult ivScanResult) {
 
     /**
      * Adjusts expandedLevelSeekbar and expandedLevelSeekbar thumbs
-     * <p/>
+     *
      * expandedLevelSeekbar is a single thumb seekbar
      * Seekbar should be max at possible Pokemon level at trainer level 40.
      * Thumb should be placed at current Pokemon level
-     * <p/>
+     *
```
Comment (Blaisorblade) [inline app/src/main/java/com/kamron/pogoiv/Pokefly.java]: Those are added by reformatting and make some sense if you ever actually want to look at generated JavaDocs... if you don't like them, would you mind changing that separately?

File: app/src/main/java/com/kamron/pogoiv/Pokefly.java [hunk 2]
```diff
@@ -1071,16 +1071,17 @@ private void populateResultsBox(IVScanResult ivScanResult) {
...
      * expandedLevelSeekbarBackground is a double thumb seekbar
      * Seekbar should be max at possible Pokemon at trainer level 40
      * Thumb 1 should be marked as an orange marker and placed at the max possible Pokemon level at the current
      * trainer level
      * Thumb 2 should be invisible and placed at the max
+     * Set empty on touch listener to prevent changing values of Thumb 1
```
Comment (Blaisorblade) [inline app/src/main/java/com/kamron/pogoiv/Pokefly.java]: Maybe this should probably be an implementation comment, not part of the JavaDoc. I know the same is true for almost this whole JavaDoc, so I guess I'm not gonna insist.

File: app/src/main/java/com/kamron/pogoiv/Pokefly.java [hunk 3]
```diff
@@ -526,7 +526,9 @@ private Drawable getDrawableC(@DrawableRes int id) {
      * @return Desired color.
      */
     @SuppressWarnings("deprecation")
-    private @ColorInt int getColorC(@ColorRes int id) {
+    private
+    @ColorInt
+    int getColorC(@ColorRes int id) {
```
Comment (DJCrapsody) [inline app/src/main/java/com/kamron/pogoiv/Pokefly.java]: Seems like 'Reformat Code' did some more changes. 😕

Comment (Blaisorblade) [inline app/src/main/java/com/kamron/pogoiv/Pokefly.java]: Have you loaded the latest formatting style? I think I committed the fixed one.

Comment (DJCrapsody) [inline app/src/main/java/com/kamron/pogoiv/Pokefly.java]: sweet. I've probably missed that change.

**Comment by DJCrapsody** [comment]
Obliterated bad commits and here we go again!

File: app/src/main/java/com/kamron/pogoiv/Pokefly.java [hunk 4]
```diff
@@ -1071,28 +1071,39 @@ private void populateResultsBox(IVScanResult ivScanResult) {
 
     /**
      * Adjusts expandedLevelSeekbar and expandedLevelSeekbar thumbs
```
Comment (Blaisorblade) [inline app/src/main/java/com/kamron/pogoiv/Pokefly.java]: Checkstyle requires a dot there:

```
:app:checkstyle[ant:checkstyle] [ERROR] /home/travis/build/farkam135/GoIV/app/src/main/java/com/kamron/pogoiv/Pokefly.java:1072: First sentence of Javadoc is incomplete (period is missing) or not present. [SummaryJavadoc]
```

**Comment by DJCrapsody** [comment]
Periods, periods everywhere. . . . .

**Comment by Blaisorblade** [comment]
Ping @DJCrapsody for both @sarav's comment and the conflict.

**Comment by nahojjjen** [comment]
@sarav if you use seekbar.setenabled(false) it grays out the seekbar, I guess he ran into this http://stackoverflow.com/questions/4020881/android-disable-seek-bar-for-audio-controler

**Comment by DJCrapsody** [comment]
Already mentioned by @nahojjjen, disabled the seekbar will gray it out. The only way is to use an empty touch event.

**Comment by Blaisorblade** [comment]
@DJCrapsody Happy to merge once you fix the conflict ;-).

**Comment by DJCrapsody** [comment]
No probs. Will deconflict when I get back from work.

**Comment by sarav** [comment]
@nahojjjen thanks for the explanation.

**Comment by DJCrapsody** [comment]
Oops, shouldn't have squashed one of the commit. @Blaisorblade conflict resolved

**Comment by sarav** [comment]
Just rebase your commit to fix conflicts folks. That way you'll avoid all the unnecessary merge commits.

**Comment by Blaisorblade** [comment]
This is small enough I'm gonna squash-merge this.

**Comment by sarav** [comment]
Thank github for squash merges!

**Pull request merged**